.cg-deploy/manifests/staging/manifest-api-staging.yml

```diff
@@ -1,6 +1,6 @@
 ---
 applications:
-- name: fs-intake-api-trees-staging
+- name: open-forest-platform-api-staging
   host: fs-intake-api-trees
```

Comment:
should the host be updated as well?

Reply:
if we update the host, we'll have to reach out to eauth, pay.gov, and login, so i think it would be easier to avoid that.